### DISCUSSION TITLE
What do you recommend Instructor Training Checkout should include? Consider new ideas, changes, or things to keep.

### DISCUSSION THREAD

**Discussion by klbarnes20** [carpentries/instructor-training#1440] [checkout-feedback]
### Background
When someone wants to be a certified Carpentries Instructor, they first take Instructor Training. After that, they must complete checkout to be certified.

**[Checkout](https://carpentries.github.io/instructor-training/checkout/) consists of three steps:**

- Attend a community discussion

- Make a contribution to a Carpentries lesson (including those in the incubator) or other resources

- Do a 5 minute teaching demo 

The Core Team is working on a project that is the first systematic review and opportunity to improve the checkout process since 2018. **We want to hear from you!**

**Comment by sarasrking** [comment]
I definitely think that the teaching demo is a thing to keep. It's hard but it's the best way to really get a handle on the process and getting some teaching practice. There might be room for more options around contributing on GitHub though, to make sure that the contributions are actually helping and not creating more work for maintainers - perhaps being more explicit about the kinds of things people can do to contribute could help. I think participating in a discussion on GitHub could be one way, picking up any editing or spell-check issues, things like that. When I did my checkout I figured that the point was to use GitHub for something (anything!), as I'd never seen it before. Obviously for heavy users of GitHub that doesn't feel like a challenge - maybe there could be two streams, one for newbies and one for more experienced GitHub users.
